languages/csharp/exercises/concept/structs/.docs/after.md

```diff
@@ -0,0 +1,37 @@
+C# `struct`s are closely related `class`s. They have state and behavior. They can have the same kinds of members, constructors, methods, fields, properties, etc fields and properties can be simple types, `struct`s and reference types. `struct`s observe the same rules about scope, read/write rules and access levels.
+
+One of the main things to remember is that when one struct is assigned to a variable or passed as a parameter the values are copied across so changes to he original variable will not affect the copied one and vice versa. In summary, `struct`s are **value types**.
+
+This [article][structs-patterns] discusses the differences between `struct`s and `class`s. You will see from the article that `struct`s tend to be lightweight and [immutable][structs-immutable] although this guidance is not enforced by the compiler or runtime.
+
+There are a couple of things that you will come up against (and about which the compiler will remind you):
+
+1. Members of a `struct` cannot be initialized inline.
+2. A `struct` cannot be inherited
+
+As a result of point 1 above there is no way for the developer of a `struct` to prevent invalid instances coming into existence.
+
+#### Common structs
+
+In most code the most commonly used library `struct` is the [`DateTime`][date-time] type.
```

Comment:
I'd argue that the most commonly used `struct` is actually `Int32` :) But the example does work here.

Reply:
No - I'll change it.  I'll say something about all the primitives being structs.

Reply:
Great!

Reply:
On reflection, as the `Int32` struct, in line with the other primitives, has a member of type `int` I don't think you can simply say the primitives are structs.  I now talk about there being a close relationship between primitives and structs.